src/SFML.Graphics/RenderWindow.cs

```diff
@@ -184,13 +184,13 @@ public override void SetTitle(string title)
         /// <param name="height">Icon's height, in pixels</param>
         /// <param name="pixels">Array of pixels, format must be RGBA 32 bits</param>
         ////////////////////////////////////////////////////////////
-        public override void SetIcon(uint width, uint height, byte[] pixels)
+        public override void SetIcon(uint width, uint height, Span<byte> pixels)
         {
             unsafe
             {
-                fixed (byte* PixelsPtr = pixels)
+                fixed (byte* ptr = pixels)

```

Comment:
I'm personally fine with the changes to `ptr` vs `PixelPtr` everywhere, but that is purely semantics and it might be beneficial to actually create some coding style rules for the project that we don't currently have and this would be relevant to that discussion.